modules/general_functions.py

```diff
@@ -656,3 +656,25 @@ def find_next(iterable: Iterable[T], cond: Callable[[T], bool]) -> T:
         return next(i for i in iterable if cond(i))
     except StopIteration:
         raise ValueError("Value not found in iterable.")
+
+
+def check_if_sum_spectrum_is_selected(directory):
+    """
+    Check if the sum spectrum is selected on the GUI.
```

Comment:
The docstring doesn't match what the function does at all. There is no GUI involved here.

Reply:
Same goes for the function name, a more exact name would be something like `check_for_sum_spectra_files(directory)`

Reply:
Pedantry. It does not **literally** check it from the GUI but it does it implicitly. However, I'll change that.